opencti-platform/opencti-front/src/private/components/settings/users/UserCreation.jsx

```diff
@@ -90,6 +107,13 @@ const UserCreation = ({ paginationOptions, defaultGroupsQueryRef }) => {
     delete finalValues.confirmation;
     delete finalValues.user_confidence_level_enabled;
 
+    if (serviceAccountFeatureFlag && finalValues.user_service_account) {
+      delete finalValues.password;
+      if (delete finalValues.user_email === '') {
```

Comment:
```suggestion
      if (finalValues.user_email === '') {
```
I think delete here is too much, sorry for that :)